config/routes.rb

```diff
@@ -8,4 +8,5 @@
   end
   get "/auth/github", as: :github_login
   get "/auth/github/callback", to: "sessions#create"
+  get "/start_auth", to: 'github#github_start'
```

Comment:
Not sure that you need this route and the github controller. Can you explain why you are using these?